apps/web/src/components/FeatureFlag.tsx

```diff
@@ -64,7 +64,7 @@ const HistoryButton = ({ flagValueId }: { flagValueId: string }) => {
             sideOffset={5}
           >
             {isLoading && <LoadingSpinner />}
-            {data !== null && (
+            {data !== undefined && (
```

Comment:
could it be both? maybe just do

```suggestion
            {data != null && (
```

Reply:
Nah, it can only be `undefined` or the value we expect. We had the non-strict checks here before, so `data != null` also catched the case where `data` would be `undefined`